uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_hw/src/uwrt_mars_rover_drivetrain_hw_actuator_interface.cpp

```diff
@@ -0,0 +1,187 @@
+#include "uwrt_mars_rover_drivetrain_hw/uwrt_mars_rover_drivetrain_hw_actuator_interface.hpp"
+
+namespace uwrt_mars_rover_drivetrain_hw {
+LifecyleNodeCallbackReturn UWRTMarsRoverDrivetrainHWActuatorInterface::on_init(
+    const hardware_interface::HardwareInfo& actuator_info) {
+  RCLCPP_DEBUG(rclcpp::get_logger("UWRTMarsRoverDrivetrainHWActuatorInterface"), "Drivetrain Actuator Initializing...");
+
+  if (hardware_interface::ActuatorInterface::on_init(actuator_info) != LifecyleNodeCallbackReturn::SUCCESS) {
+    return LifecyleNodeCallbackReturn::ERROR;
+  }
+
+  // Validate that parsed urdf information matches expected structure
+  if (info_.joints.size() != NUM_JOINTS) {
+    RCLCPP_FATAL_STREAM(rclcpp::get_logger("UWRTMarsRoverDrivetrainHWActuatorInterface"),
+                        "'" << info_.name.c_str() << "' has " << info_.joints.size() << " joints. 1 expected.");
+    return LifecyleNodeCallbackReturn::ERROR;
+  }
+  const hardware_interface::ComponentInfo& joint = info_.joints[0];
+  if (joint.state_interfaces.size() != NUM_STATE_INTERFACES) {
+    RCLCPP_FATAL_STREAM(rclcpp::get_logger("UWRTMarsRoverDrivetrainHWActuatorInterface"),
+                        "Joint '" << joint.name.c_str() << "' has " << joint.command_interfaces.size()
```

Comment:
aren't you checking State interfaces here not command interfaces ..... they are the number of state and command so not an issue ... but just curious